### PR TITLE
feat: Swap api

### PR DESCRIPTION
**Pull request by nuel77**
- queries and extrincs to interact with the swap pallet of polkadex.

### CODE REVIEW

File: packages/polkadex-api/src/modules/swap.test.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    expect(res).toEqual(expect.any(Number));
+  });
+
+  test("quote tokens for exact tokens", async () => {
+    const res = await swapApi.quotePriceTokensForExactTokens(
+      "PDEX",
+      "95930534000017180603917534864279132680",
+      1000000000000000
+    );
+    console.log(res);
```
Comment (minikas) [inline packages/polkadex-api/src/modules/swap.test.ts:43]: remove console.log

File: packages/polkadex-api/src/modules/swap.test.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    expect(res[0].lpToken).toEqual(expect.any(String));
+  });
+
+  test("quote exact tokens for tokens", async () => {
+    const res = await swapApi.quotePriceExactTokensForTokens(
+      "PDEX",
+      "95930534000017180603917534864279132680",
+      100000000000000
+    );
+    console.log(res);
```
Comment (minikas) [inline packages/polkadex-api/src/modules/swap.test.ts:33]: remove console.log

File: packages/polkadex-api/src/modules/swap.ts [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+  public async swapExactTokensForTokensTx(
+    path: string[],
+    amountIn: number,
+    amountOutMin: number,
+    toAddress: string,
+    keepAlive = true
+  ): Promise<SubmittableExtrinsic> {
+    const assetPath = path.map((asset) => this.createAssetIdEnum(asset));
+    const amtIn = toPlanck(amountIn, this.chainDecimals).toFixed();
+    console.log("amountIn", amtIn);
```
Comment (minikas) [inline packages/polkadex-api/src/modules/swap.ts:184]: remove console.log

**Review by minikas — CHANGES_REQUESTED** [review]

File: packages/polkadex-api/src/modules/swap.test.ts (hunk 1 above)
Comment (nuel77) [reply packages/polkadex-api/src/modules/swap.test.ts]: removed

File: packages/polkadex-api/src/modules/swap.test.ts (hunk 2 above)
Comment (nuel77) [reply packages/polkadex-api/src/modules/swap.test.ts]: removed

**Review by minikas — APPROVED** [review]
LGTM

**Pull request merged**